colossalai/engine/schedule/_non_pipeline_schedule.py

```diff
@@ -16,10 +17,32 @@ class NonPipelineSchedule(BaseSchedule):
     to update the parameters if it is in training mode.
 
     Args:
-        batch_data_process_func (Callable, optional): The preprocessing function which receives a batch of data,
+        batch_data_process_func (Callable, optional): The preprocessing function which receives a batch of data
```

Comment:
The argument name should be data_process_func.

Reply:
Ok